quarkus/pom.xml

```diff
@@ -31,10 +31,11 @@
     <packaging>pom</packaging>
 
     <properties>
-        <quarkus.version>1.2.1.Final</quarkus.version>
+        <quarkus.version>999-SNAPSHOT</quarkus.version>
```

Comment:
Don't think we should require folks to build their own Quarkus to use Keycloak should we?